### DISCUSSION TITLE
Can't open copy of Keepass DB

### DISCUSSION THREAD

**Discussion by SlaveOfCats** [Kunzisoft/KeePassDX#1388] [Q&A]
I made a copy of my Keepass DB by using the "Save a copy to..." menu selection in KeepassDX. Then renamed the original Keepass DB (from outside of KeepassDX). KeepassDX will not open either copy anymore. 

I made the copy before fooling around with the original DB just in case doing that would cause a problem, but the copy seems to be useless. Am I missing something about how to use the copy as a backup? Is there a way to open either copy at this point?

**Comment by J-Jamet** [comment]
"Save a copy to..." allows to save the database data stream in a new stream of the selected file provider. If you select a local file, it will be saved to a local file. If you select a file in a cloud application, it is possible that the cloud application uses an internal cache or other storage format. So just check where you are saving the data.
